File: backend/ImpactAPI/Models/Entities/Currency.cs
namespace ImpactAPI.Models.Entities
{
    public class Currency
    {

        public string Code { get; set; } = default!;
        public string Codein { get; set; } = default!;
        public string Name { get; set; } = default!;
        public decimal Bind { get; set; } = default!;
        public decimal Ask { get; set; } = default!;
    }
}

File: backend/products-api/src/entities/Currency.ts
export interface CurrencyProps {
  code: string;
  codein: string;
  name: string;
  high: string;
}

export class Currency {
  constructor(private props: CurrencyProps) {}
  get code() {
    return this.props.code;
  }
  get codein() {
    return this.props.codein;
  }
  get name() {
    return this.props.name;
  }
  get high() {
    return this.props.high;
  }
}
